### DISCUSSION TITLE
merging dataArrays without the need for a DataSet

### DISCUSSION THREAD

**Discussion by dorak88783** [pydata/xarray#9323] [General]
I have a list of dataarrays that I'd like to merge. In theory `xr.merge` does the job, but it will return a Dataset and refuses to work on unnamed dataArrays.

Example:

```
x = xr.DataArray(
    [[1.0, 2.0], [3.0, 5.0]],
    dims=("lat", "lon"),
    coords={"lat": [35.0, 40.0], "lon": [100.0, 120.0]},
    name="var1"
)
x1=x.where(x.lat==35)
x2=x.where(x.lon==100)
xr.merge([x1, x2])[x.name]
```

This works, but it's a bit silly to have to include the `[x.name]` at the end to get the dataArray out. Now if I remove the `name="var1"` line, it will not work.

How do I get the exact same functionality of `xr.merge`, just applied to lists of dataArrays, without any regards for the `.name` property?

**Comment by dcherian** [comment]
See https://docs.xarray.dev/en/stable/user-guide/combining.html

**Reply by dorak88783** [reply]
> See https://docs.xarray.dev/en/stable/user-guide/combining.html

I think I really tried all the options listed there, but none will work. Which one should I use to make my example above (without the `name="var1"` line) work?

**Reply by TomNicholas** [reply]
What do you want the result of your combining operation to be? At the moment saying you want to merge them but you want them to remain unnamed is not possible - it's like saying you want to put two values in a dictionary without choosing names for keys to store them under - a dictionary doesn't allow that by definition.
